lib/streamlit/elements/arrow_altair.py

```diff
@@ -63,12 +63,21 @@
 
 
 class ChartType(Enum):
-    AREA = "area"
-    BAR = "bar"
-    LINE = "line"
-
-
-COLOR_LEGEND_SETTINGS = dict(titlePadding=0, offset=10, orient="bottom")
+    AREA = {"mark_type": "area"}
+    BAR = {"mark_type": "bar"}
+    LINE = {"mark_type": "line"}
+    SCATTER = {"mark_type": "circle"}
+
+
+# Color and size legends need different title paddings in order for them
+# to be vertically aligned.
+# NOTE: I don't think it's possible to *perfectly* align the size and
+# color legends in all instances, since the "size" circles vary in size based
+# on the data, and their container is top-aligned with the color container. But
+# through trial-and-error I found this value to be a good enough middle ground.
+# See e2e/scripts/st_arrow_scatter_chart.py for some alignment tests.
+COLOR_LEGEND_SETTINGS = dict(titlePadding=5, offset=5, orient="bottom")
+SIZE_LEGEND_SETTINGS = dict(titlePadding=0.5, offset=5, orient="bottom")
```

Comment:
I think same here.

Reply:
I moved all I could to CustomTheme.tsx, but I need to leave the titlePadding here in Python because there's no way to set a different value for one encoding (i.e. `color`) vs another encoding (`size`).

Reply:
Actually, I just realized that by moving this we broke some existing behavior.

When you create an Altair/Vega chart by hand, we used to (1) apply Streamlit colors, but (2) NOT move the legend to the bottom. With this change, we now DO move the legend to the bottom.

I need to double-check whether `#2` was the desired behavior or just a bug.